### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by practicalswift**

### CODE REVIEW

**Comment by gottesmm** [comment]
Can you split these up into manageable pull requests? I think it is unreasonable for one person to review all of these. I would suggest no more than 10 commits per pull request (I would normally say much smaller than that, but these are just typos).

File: docs/archive/LangRef.html [hunk 1]
```diff
@@ -72,7 +72,7 @@ <h4 id="brace-item-list">Brace Enclosed Items</h4>
   <p>The brace item list provides a sequencing operation which evaluates the
   members of its body in order.  Function bodies and the bodies of control
   flow statements use braces.  Also, the <a
-    href="#decl-top-level">source file</a> itself is effectively a
+    href="#decl-top-level">source file</a> itself is effectivelyy a
```
Comment (dtorres) [inline docs/archive/LangRef.html]: That doesn't look like an effective fix

Comment (jtbandes) [inline docs/archive/LangRef.html]: looks like a case of `s/effectivel/effectively/` :)

**Comment by jtbandes** [comment]
IMO these should be squashed into one commit.

**Comment by dduan** [comment]
I imagine there's a script involved in this (I thought about writing one myself, you beat me to it :)). I think adding a build flag to to do spellchecking is a decent idea.

File: docs/archive/LangRef.html (hunk 1 above)
Comment (practicalswift) [inline docs/archive/LangRef.html]: @dtorres Good catch! Thanks for reporting! :-)

**Comment by practicalswift** [comment]
@gottesmm Sure, no problem! :-)

**Pull request closed without merging**